server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/aeternity/AeternityExtension.java

```diff
@@ -0,0 +1,106 @@
+/*************************************************************************************
+ * Copyright (C) 2014-2019 GENERAL BYTES s.r.o. All rights reserved.
+ *
+ * This software may be distributed and modified under the terms of the GNU
+ * General Public License version 2 (GPL2) as published by the Free Software
+ * Foundation and appearing in the file GPL2.TXT included in the packaging of
+ * this file. Please note that GPL2 Section 2[b] requires that all works based
+ * on this software must also be made publicly available under the terms of
+ * the GPL2 ("Copyleft").
+ *
+ * Contact information
+ * -------------------
+ *
+ * GENERAL BYTES s.r.o.
+ * Web      :  http://www.generalbytes.com
+ *
+ ************************************************************************************/
+package com.generalbytes.batm.server.extensions.extra.aeternity;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.StringTokenizer;
+
+import com.generalbytes.batm.common.currencies.CryptoCurrency;
+import com.generalbytes.batm.common.currencies.FiatCurrency;
+import com.generalbytes.batm.server.extensions.AbstractExtension;
+import com.generalbytes.batm.server.extensions.ICryptoAddressValidator;
+import com.generalbytes.batm.server.extensions.ICryptoCurrencyDefinition;
+import com.generalbytes.batm.server.extensions.IRateSource;
+import com.generalbytes.batm.server.extensions.IWallet;
+import com.generalbytes.batm.server.extensions.extra.bitcoin.sources.coingecko.CoinGeckoRateSource;
+import com.kryptokrauts.aeternity.sdk.util.EncodingUtils;
+
+public class AeternityExtension extends AbstractExtension {
+    private static final ICryptoCurrencyDefinition DEFINITION = new AeternityDefinition();
+    public static final String CURRENCY = CryptoCurrency.AE.getCode();
+
+    @Override
+    public String getName() {
+        return "BATM Aeternity extra extension";
+    }
+
+    @Override
+    public Set<String> getSupportedCryptoCurrencies() {
+    	HashSet<String> result = new HashSet<>();
+        result.add(CURRENCY);
+        return result;
+    }
+    
+    @Override
+    public IWallet createWallet(String walletLogin) {
+    	System.out.println("wallet login:" + walletLogin);       
```

Comment:
Please remove all System.out calls in your code.

Reply:
ok, will remove those